app/renderer/js/pages/preference/add-certificate.ts

```diff
@@ -55,12 +63,12 @@ class AddCertificate extends BaseComponent {
 			this.serverUrl.value = '';
 		} else {
 			dialog.showErrorBox('Error', `Please, ${serverUrl === '' ?
-      'Enter an Organization URL' : 'Choose certificate file'}`);
+				'Enter an Organization URL' : 'Choose certificate file'}`);
 		}
 	}
 
-	addHandler() {
-		const showDialogOptions = {
+	addHandler(): void {
+		const showDialogOptions: OpenDialogOptions = {
```

Comment:
Using Electron.OpenDialogOptions here and removing the wrong import here should fix this. I think the import here was added automatically by the text editor. @priyank-p FYI.

Reply:
I think it's fine to leave this as is since `Electron.OpenDialogOptions` is too verbose.